packages/compass-query-bar/src/components/query-bar.tsx

```diff
@@ -317,73 +315,53 @@ export const QueryBar: React.FunctionComponent<QueryBarProps> = ({
           show={isAIInputVisible}
         />
       )}
-      <AISignInModal></AISignInModal>
     </form>
   );
 };
 
 type OwnProps = {
-  enableAIExperience: boolean;
   onApply?(query: unknown): void;
   onReset?(query: unknown): void;
 };
 
-export default withPreferences(
-  connect(
-    (
-      {
-        queryBar: { expanded, fields, applyId },
-        aiQuery,
-        atlasSignIn: { state: signInState },
-      }: RootState,
-      ownProps: OwnProps
-    ) => {
-      return {
-        expanded: expanded,
-        queryChanged: !isEqualDefaultQuery(fields),
-        valid: isQueryValid(fields),
-        applyId: applyId,
-        isAIInputVisible: aiQuery.isInputVisible,
-        enableAIQuery:
-          ownProps.enableAIExperience &&
-          // Make sure we don't show the prompt to enable ai experience while we
-          // are restoring sign in state. This operation is quick, and the check
-          // makes sure that if user is already signed in, it will get to the ai
-          // input without the need to see opt in modal
-          !['initial', 'restoring'].includes(signInState),
-      };
-    },
-    (dispatch: QueryBarThunkDispatch, ownProps: OwnProps) => {
-      return {
-        onExplain: () => {
-          dispatch(explainQuery());
-        },
-        onOpenExportToLanguage: () => {
-          dispatch(openExportToLanguage());
-        },
-        onApply: () => {
-          const applied = dispatch(applyQuery());
-          if (applied === false) {
-            return;
-          }
-          ownProps.onApply?.(applied);
-        },
-        onReset: () => {
-          const reset = dispatch(resetQuery());
-          if (reset === false) {
-            return;
-          }
-          ownProps.onReset?.(reset);
-        },
-        onShowAIInputClick: () => {
-          dispatch(showInput());
-        },
-        onHideAIInputClick: () => {
-          dispatch(hideInput());
-        },
-      };
-    }
-  )(QueryBar),
-  ['enableAIExperience'],
-  React
-);
+export default connect(
```

Comment:
This looks like a lot of lines changed, but I just removed the `withPreferences` wrapper that was added previously as we don't need to derive `enableAIQuery` from `signInState` anymore, so no need for preferences to be available in the `mapState` function